DBMS module/flow_sequence_diag.md

```diff
@@ -1,7 +1,7 @@
 # Data Flow Diagram
 
 
-<img width="650" height="700" src="../images/flow-sequensce.png" alt="Flow sequence diagram" border= "7" style= "display: block; margin-left: auto; margin-right: auto; ">
+<kbd><img width="650" height="700" src="../images/flow-sequensce.png" alt="Flow sequence diagram" style="border: 4px solid black" style= "display: block; margin-left: auto; margin-right: auto; "></kbd>
```

Comment:
@96RadhikaJadhav May I know the significance of using ```<kbd>``` here :thinking:

Reply:
When I searched it on stack overflow, in one of the answers I found this tag. So tried applying it.